docs/dsp-tools-usage.md

```diff
@@ -254,61 +254,45 @@ In order to upload data incrementally the procedure described [here](dsp-tools-x
 
 
 
-## Start a DSP-stack on your local machine (for DaSCH-internal use only)
+## Start a DSP stack on your local machine 
+
+DSP-API is the heart of the DaSCH software - it is a server application for storing data from the Humanities.
+DSP-APP is our generic user interface, i.e. the viewer to look at the data. It's a server application, too.
 
 For testing purposes, it is sometimes necessary to run DSP-API and DSP-APP on a local machine. But the startup 
 and shutdown of API and APP can be complicated: Both repos need to be cloned locally, a `git pull` has to be executed 
-from time to time to stay up to date, and then there are several commands for each repository to remember. 
-
-Another challenge is the software that DSP depends upon: JDK, node, npm, Angular, etc. should be kept up to date. And
-it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. An non-developer can quickly get lost
-in this jungle. 
-
-That's why dsp-tools offers some commands to facilitate the handling of API and APP. These commands 
-
- - clone the repos to `~/.dsp-tools`, and keep them up to date.
- - check every time if the dependencies are up to date, and give you advice how to update them, if necessary.
- - pass on the right commands to APP and API, even if the correct usage of these commands changes over time.
- - make sure that the repos don't get cluttered with old files over time.
- - log their activity in `~/.dsp-tools`, so you can check the logs for troubleshooting, if necessary.
-
-The only requirements for these commands are:
-
- - the initial installation of all software that you accomplished when you started working at DaSCH
- - Docker must be running (for DSP-API only)
-
-Please note that these commands were developed for DaSCH-internal use only. They only work on Macs that have the 
-required software installed that makes it possible to run the API and APP. We don't offer support or troubleshooting 
-for these commands.
-
+from time to time to stay up to date, and then there are several commands for each repository to remember. And your 
+local clone of the API might get cluttered with old files over time.
 
-### Start DSP-API
+Another challenge is the software that DSP depends on, and that should be kept up to date: JDK, node, npm, Angular, 
+etc. And it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. A non-developer can quickly 
+get lost in this jungle. 
```

Comment:
I recommend to remove this whole section from the documentation. It is not the right place to inform the user about decisions that were made regarding the setup. I assume the user just wants to know what to do - as briefly and simple as possible. 
We also have the problem in DSP-API that different "users" are targeted by the same documentation (end-users, internal/external software developers). This resulted in a messy documentation over time. We can brainstorm about this topic, if you like. It's not easy to set this up alone.

Reply:
I principally agree with you, but there is one reason why I am reluctant: It is now a years-old practice to install all this software on the machines of new employees, and to execute commands in the repositories. For me it was an "aha moment" to learn the distinction between building from the repositories (needs dependencies) versus running Docker containers (no dependencies needed). That's the reason why I feel the need to clearly document this change. If I just communicate it internally in a chat, it will be forgotten after some days/weeks. 

But I'm open for brainstorming about better options.